server/buttonAlerts.js

```diff
@@ -81,7 +82,7 @@ async function handleValidRequest(button, numButtonPresses) {
         button.client.responderPhoneNumbers,
         button.phoneNumber,
         t('alertUrgent', { lng: button.client.language, numButtonPresses: currentSession.numButtonPresses.toString() }),
-        `${helpers.getAlertTypeDisplayName(ALERT_TYPE.BUTTONS_URGENT)} Alert:\n${button.displayName.toString()}`,
+        `${helpers.getAlertTypeDisplayName(ALERT_TYPE.BUTTONS_URGENT, button.client.language)} Alert:\n${button.displayName.toString()}`,
       )
```

Comment:
I think you might also need to also pass `t` as an argument for this function.

Reply:
I agree.